src/main/java/org/taktik/icure/logic/impl/filter/service/ServiceByHcPartyTagCodeDateFilter.java

```diff
@@ -54,18 +54,21 @@ public void setSessionLogic(ICureSessionLogic sessionLogic) {
 		try {
             String hcPartyId = filter.getHealthcarePartyId() != null ? filter.getHealthcarePartyId() : getLoggedHealthCarePartyId();
             HashSet<String> ids = null;
+            List<String> patientSFKList = filter.getPatientSecretForeignKey() != null ?
```

Comment:
`filter.getPatientSecretForeignKey()` better to extract to variable before check